dnf5/main.cpp

```diff
@@ -738,6 +738,14 @@ int main(int argc, char * argv[]) try {
                 }
             }
             std::cerr << ex.what() << _(". Add \"--help\" for more information about the arguments.") << std::endl;
+            if (auto * unknown_arg_ex = dynamic_cast<libdnf5::cli::ArgumentParserUnknownArgumentError *>(&ex)) {
+                if (unknown_arg_ex->get_command() == "dnf5") {
+                    std::cout << fmt::format(
+                                     "It could be a command provided by a plugin, try: dnf install dnf5-command({})",
```

Comment:
The warning is currently shown even when invalid parameter is passed to `dnf5`:
```
[root@c3c9bc2f0fde dnf5]# dnf5 --param
Unknown argument "--param" for command "dnf5". Add "--help" for more information about the arguments.
It could be a command provided by a plugin, try: dnf install dnf5-command(--param)
```

Reply:
Ah thanks. Now I'm also checking that the argument doesn't begin with "-".